task_sdk/tests/execution_time/test_supervisor.py

```diff
@@ -250,18 +258,40 @@ def watched_subprocess(self, mocker):
                 GetConnection(conn_id="test_conn"),
                 b'{"conn_id":"test_conn","conn_type":"mysql"}',
                 "connections.get",
-                "test_conn",
+                ("test_conn",),
                 ConnectionResult(conn_id="test_conn", conn_type="mysql"),
                 id="get_connection",
             ),
             pytest.param(
                 GetVariable(key="test_key"),
                 b'{"key":"test_key","value":"test_value"}',
                 "variables.get",
-                "test_key",
+                ("test_key",),
                 VariableResult(key="test_key", value="test_value"),
                 id="get_variable",
             ),
+            pytest.param(
+                DeferTask(
+                    state="deferred",
+                    kwargs={},
+                    next_method="execute_callback",
+                    timeout=None,
+                    type="DeferTask",
+                ),
+                b'{"state":"deferred","classpath":"my-class-path","kwargs":{},"created_date":"2024-10-31T12:00:00Z","next_method":"execute_callback","timeout":null}',
```

Comment:
We don't need send the message -- so we can remove this line or keep it as `b""` since we don't send any msg

Reply:
Yeah, made it `b""`